library/src/window_impl.cxx

```diff
@@ -282,7 +283,7 @@ int window_impl::getHeight() const
 //----------------------------------------------------------------------------
 window& window_impl::setSize(int width, int height)
 {
-  this->Internals->RenWin->SetSize(width, height);
+  this->Internals->RenWin->GetInteractor()->UpdateSize(width, height);
```

Comment:
What if there is no interactor ?

Reply:
There's always an interactor now (except for `createNone()`) but in this case there is no window either.

Reply:
assert then